broccoli/build-info.js

```diff
@@ -34,9 +34,9 @@ function buildInfo(options) {
 function buildGitInfo(root) {
   let info = gitRepoInfo(root);
   return {
-    sha: process.env.TRAVIS_COMMIT || info.sha,
-    branch: process.env.TRAVIS_BRANCH || info.branch,
-    tag: process.env.TRAVIS_TAG || info.tag,
+    sha: info.sha,
+    branch: process.env.TRAVIS_BRANCH || info.branch, // Travis builds are always detached
```

Comment:
What is `info.branch` when detached? 

I kinda feel like we may want to put all the `TRAVIS` smarts in `buildFromParts` directly. What do you think?

Reply:
`info.branch` when detached is null.

Reply:
@rwjblue Can you restate or clarify "I kinda feel like we may want to put all the TRAVIS smarts in buildFromParts directly. What do you think?"?

Reply:
Basically, just that this could be `branch: info.branch` and down below where all the other travis detection logic.

Reply:
👍